ninja-core/src/main/java/ninja/Bootstrap.java

```diff
@@ -208,6 +209,7 @@ protected void configure() throws Exception {
             @Override
             protected void configure() {
                 bind(Ninja.class).to(ninjaClass).in(Singleton.class);
+                Multibinder.newSetBinder(binder(), ParamParser.class);
```

Comment:
Not sure, but either we add the binder here or in NinjaBaseModule - but not in both... I guess NinjaBaseModule is the right place...